packages/rancher-monitoring/rancher-pushprox/charts/values.yaml

```diff
@@ -5,11 +5,6 @@
 # Default image containing both the proxy and the client was generated from the following Dockerfile
 # https://github.com/prometheus-community/PushProx/blob/eeadbe766641699129920ccfaaaa30a85c67fe81/Dockerfile#L1-L15
 
-# Note: if using a cloud provider, the nodes that contain the PushProxy client must allow the node(s) that contain(s)
```

Comment:
I don't think this note is accurate or necessary anymore. Might have been a remnant of the initial design of the chart.